crates/libs/metadata/src/reader/blob.rs

```diff
@@ -70,6 +70,14 @@ impl<'a> Blob<'a> {
             String::from_utf16_lossy(slice)
         }
     }
+    pub fn read_bool(&mut self) -> bool {
+        // A bool is specified as "a single byte with value 0 (false) or 1 (true)".
```

Comment:
ECMA-335, II.7.4 Native data types ([PDF](https://www.ecma-international.org/wp-content/uploads/ECMA-335_6th_edition_june_2012.pdf) page 125), states:
> Boolean. 4-byte integer value where any non-zero value represents TRUE, and 0 represents 
FALSE.

Reply:
Sorry, I should have been clear, this is a custom attribute, ECMA-335, II.23.3 (page 268).

> If the parameter kind is simple (first line in the above diagram) ( bool, char, float32, float64, int8, int16, int32, int64, unsigned int8, unsigned int16, unsigned int32 or unsigned int64) then the 'blob' contains its binary value (Val). (A bool is a single byte with value 0 (false) or 1 (true); char is a two-byte Unicode character; and the others have their obvious meaning.) This pattern is also used if the parameter kind is an enum -- simply store the value of the enum's underlying integer type.

Reply:
Ah, I should've paid closer attention--this is a blob. 👍